{
  "uid" : "6f4d032c78c4c9fd",
  "name" : "table merge in sync",
  "fullName" : "test_pipeline.TestPipeline#test_merge_sync",
  "historyId" : "edfd051c418b47aafa88b402eedbe088",
  "time" : {
    "start" : 1666090871129,
    "stop" : 1666090871139,
    "duration" : 10
  },
  "status" : "broken",
  "statusMessage" : "NameError: name 'source_name' is not defined",
  "statusTrace" : "self = <test.test_pipeline.TestPipeline object at 0x7f686a42eb90>\n\n    @allure.title(\"table merge in sync\")\n    def test_merge_sync(self):\n        p = make_new_pipeline(f\"merge_sync_{random_str()}\")\n        p.dag.jobType = JobType.sync\n>       p1 = p.readFrom(f\"{source_name}.test\")\nE       NameError: name 'source_name' is not defined\n\ntest_pipeline.py:136: NameError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "pipeline"
  }, {
    "name" : "suite",
    "value" : "test_pipeline"
  }, {
    "name" : "subSuite",
    "value" : "TestPipeline"
  }, {
    "name" : "host",
    "value" : "125898ca5794"
  }, {
    "name" : "thread",
    "value" : "881-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_pipeline"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "6f4d032c78c4c9fd.json",
  "parameterValues" : [ ]
}